05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,673 @@
+
+If a dataset had WT0 for the main weight, 20 bootstrap weights indicated WT1, WT2, ..., WT20, and $\alpha=.02$, use the following syntax:
+
+```r
+des_bs <- dat %>%
+  as_survey_rep(weights = WT0, repweights= num_range("WT", 1:20),
+                type="bootstrap", mse=TRUE, scale=.02)
+
+```
+
+Note that the scale is usually provided in documentation and is a constant, so is not provided as a variable in the tibble.
+
+#### Example {-}
+
+Returning to the api example, bootstrap weights were constructed for a one cluster design. 50 replicate weights were created on a dataset `apiclus1_slim` which has some familiar variables including cds, dnum, fpc, and pw but now additionally includes bootstrap weights pw1, ..., pw50. The scale $(\alpha)$ is $15/(14*49)=0.02186589$
+
+```{r}
+#| label: genbs
+#| include: FALSE
+apiclus1_slim <- 
+  apiclus1 %>%
+  as_tibble() %>%
+  arrange(dnum) %>%
+  select(cds, dnum, fpc, pw)
+
+set.seed(662152)
+apibw <- bootweights(psu=apiclus1_slim$dnum, strata=rep(1, nrow(apiclus1_slim)), fpc=apiclus1_slim$fpc, replicates=50)
+
+bwmata <- apibw$repweights$weights[apibw$repweights$index, ]*apiclus1_slim$pw
+
+apiclus1_slim <- bwmata %>%
+  as.data.frame() %>%
+  set_names(str_c("pw", 1:50)) %>%
+  cbind(apiclus1_slim) %>%
+  as_tibble() %>%
+  select(cds, dnum, fpc, pw, everything())
+```
+
+```{r}
+#| label: bsexamp
+apiclus1_slim
+
+des_api1_bs <- apiclus1_slim %>% 
+  as_survey_rep(weights=pw, repweights=pw1:pw50, type="bootstrap", 
+                scale=0.02186589, mse=TRUE)
+
+des_api1_bs
+
+summary(des_api1_bs)
+```
+
+As with other replicate design objects, when printing the object or looking at the summary, the replicate weights are provided along with the data variables.
+
+## Understanding survey design documentation
+
+SRS, stratified, and clustered designs are the backbone of sampling designs and the features are often combined in one design. Additionally, rather than using SRS for selection, other sampling mechanisms are commonly used such as probability proportional to size (PPS), systematic sampling, or selection with unequal probabilities which are briefly described here. In PPS sampling, a size measure is constructed for each unit - perhaps the population of the PSU or the number of occupied housing units, and then units with larger size measures are more likely to be sampled. Systematic sampling is commonly used to ensure representation across a population. Units are sorted by a feature and then every $k$ units are selected from a random start point so the sample is spread across the population. In addition to PPS, other unequal probabilities of selection may be used. As an example, in a establishment survey, a establishment that recently participated have a reduced chance of selection in a subsequent round of the same survey to reduce the burden on the establishment. To learn more about sampling designs, refer to @valliant2013practical, @cox2011business, @cochran1977sampling, and @deming1991sample.
```

Comment:
```suggestion
SRS, stratified, and clustered designs are the backbone of sampling designs and the features are often combined in one design. Additionally, rather than using SRS for selection, other sampling mechanisms are commonly used such as probability proportional to size (PPS), systematic sampling, or selection with unequal probabilities which are briefly described here. In PPS sampling, a size measure is constructed for each unit - perhaps the population of the PSU or the number of occupied housing units, and then units with larger size measures are more likely to be sampled. Systematic sampling is commonly used to ensure representation across a population. Units are sorted by a feature and then every $k$ units are selected from a random start point so the sample is spread across the population. In addition to PPS, other unequal probabilities of selection may be used. As an example, in a study of establishments that conducts a survey every year, an establishment that recently participated (e.g., participated last year) has a reduced chance of selection in a subsequent round to reduce the burden on the establishment. To learn more about sampling designs, refer to @valliant2013practical, @cox2011business, @cochran1977sampling, and @deming1991sample.
```